### PR TITLE
Add pronto

### PR DESCRIPTION
**Pull request by magneland**

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -1,5 +1,15 @@
 require 'simplecov'
-SimpleCov.start 'rails'
+require 'simplecov-lcov'
+
+SimpleCov::Formatter::LcovFormatter.config.report_with_single_file = true
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
+  SimpleCov::Formatter::LcovFormatter,
```
Comment (magneland) [inline test/test_helper.rb]: Layout/IndentFirstArrayElement: Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.

File: test/test_helper.rb [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+  SimpleCov::Formatter::HTMLFormatter,
```
Comment (magneland) [inline test/test_helper.rb]: Style/TrailingCommaInArrayLiteral: Avoid comma after the last item of an array.

File: test/test_helper.rb [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+])
```
Comment (magneland) [inline test/test_helper.rb]: Layout/IndentFirstArrayElement: Indent the right bracket the same as the first position after the preceding left parenthesis.

File: test/test_helper.rb [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+SimpleCov.start do
+  add_filter(/^\/test\//) # For Minitest
```
Comment (magneland) [inline test/test_helper.rb]: Style/RegexpLiteral: Use `%r` around regular expression.

**Comment by magneland** [comment]
[Add auto-generated comments to PR](https://trello.com/c/vK4ymMYf/29-add-auto-generated-comments-to-pr)

File: test/test_helper.rb (hunk 1 above)
Comment (magneland) [reply test/test_helper.rb]: This bot was hooked up to my GitHub. Will fix.

File: test/test_helper.rb [hunk 5]
```diff
@@ -1,5 +1,16 @@
 require 'simplecov'
-SimpleCov.start 'rails'
+require 'simplecov-lcov'
+
+SimpleCov::Formatter::LcovFormatter.config.report_with_single_file = true
+SIMPLECOV_FORMATTERS = [
```
Comment (pronto-saurus) [inline test/test_helper.rb]: Style/MutableConstant: Freeze mutable objects assigned to constants.

File: test/test_helper.rb [hunk 6]
```diff
@@ -1,5 +1,16 @@
...
+  SimpleCov::Formatter::LcovFormatter,
+  SimpleCov::Formatter::HTMLFormatter
+]
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new(SIMPLECOV_FORMATTERS)
```
Comment (pronto-saurus) [inline test/test_helper.rb]: Metrics/LineLength: Line is too long. [84/80]

Comment (magneland) [reply test/test_helper.rb]: Fixed.

File: test/test_helper.rb (hunk 5 above)
Comment (magneland) [reply test/test_helper.rb]: Fixed.

File: test/test_helper.rb (hunk 4 above)
Comment (magneland) [reply test/test_helper.rb]: Fixed.

File: test/test_helper.rb (hunk 3 above)
Comment (magneland) [reply test/test_helper.rb]: Fixed.

File: test/test_helper.rb (hunk 2 above)
Comment (magneland) [reply test/test_helper.rb]: Fixed.

File: test/test_helper.rb (hunk 1 above)
Comment (magneland) [reply test/test_helper.rb]: Fixed.

**Pull request merged**